packages/boot/index.js

```diff
@@ -0,0 +1,8 @@
+// Copyright IBM Corp. 2013,2018. All Rights Reserved.
+// Node module: @loopback/boot
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+const nodeMajorVersion = +process.versions.node.split('.')[0];
+module.exports =
+  nodeMajorVersion >= 7 ? require('./dist/src') : require('./dist6/src');
```

Comment:
My understanding is that we will support Node 6 till we can create a PR to drop it all together from all packages because otherwise we may start seeing CI failures.